src/frontend/components/search/Search.tsx

```diff
@@ -1,52 +1,45 @@
 import React, { useState } from 'react'
-import { useDispatch } from 'react-redux'
 import styled from 'styled-components'
-import { searchWordSubmitted } from './searchSlice'
+import { searchPlaceholder } from '../../lib/index'
 import { fetchNews } from '../../api/yahooFinance'
 
-const SearchContainer = styled.div`
-  padding: 0 20px;
-
+const SearchWrapper = styled.div`
   form {
     display: flex;
+    justify-content: space-between;
   }
 
   input {
-    width: 75%;
-    margin-right: 5%;
+    width: 70%;
   }
 
   button {
     width: 20%;
   }
 `
 
-export const Search = () => {
+export default function Search() {
   const [value, setValue] = useState('')
-  const dispatch = useDispatch()
-  const disabled: boolean = !value.length
+  let [news, setNews] = useState('')
+  const isDisabled: boolean = value === '' ? true : false
```

Comment:
You don't need the ternary here.
```suggestion
  const isDisabled: boolean = value === ''
```